### ISSUE TITLE
Laravel 6 support

### ISSUE THREAD

**Issue by ronnievisser** [okaybueno/laravel-repositories#4]
Wil this package be updated to be used with L6?

**Comment by jespejoh** [comment]
Good question – we are starting to see more and more issues with the approach of using repositories with Eloquent (especially when using complex queries), so we are still evaluating whether we should keep supporting and updating this package.

**Comment by ronnievisser** [comment]
Agree! But since it’s in my project I cannot upgrade to 6 at the moment. Perhaps I should just take it out then
